crates/typst/src/math/class.rs

```diff
@@ -2,12 +2,15 @@ use unicode_math_class::MathClass;
 
 use crate::diag::SourceResult;
 use crate::foundations::{elem, Content, Packed};
-use crate::math::{LayoutMath, MathContext};
+use crate::math::{LayoutMath, Limits, MathContext};
 
 /// Forced use of a certain math class.
 ///
 /// This is useful to treat certain symbols as if they were of a different
-/// class, e.g. to make a symbol behave like a relation.
+/// class, e.g. to make a symbol behave like a relation. The class of a symbol
+/// defines spacing around the symbol, and how scripts are attached to it by
```

Comment:
If I understand it right, it isn't only the spacing. The symbol also may be moved vertically, and I think there should be more consequences. I think it's important this is not just the spacing.

See this example, where `b` letter moves down when set as `opening` class:

https://sitandr.github.io/typst-examples-book/book/basics/math/classes.html

So I have no idea how to write it in a good way. I haven't look thoroughly in the source, though.

Reply:
What about "The class of a symbol defines the way it is laid out, including spacing around it, and how its scripts are attached to by default."?

Reply:
Sounds good!